packages/react/src/auto/polaris/PolarisDateTimePicker.tsx

```diff
@@ -130,17 +158,46 @@ export const PolarisDateTimePicker = (props: {
           />
         </div>
       </Popover>
-      {props.includeTime && (
-        <TextField
-          id={props.id ? `${props.id}-time` : undefined}
-          label={props.timeLabel ?? "Time"}
-          prefix={<Icon source={ClockIcon} />}
-          autoComplete="off"
-          value={timeString}
-          onChange={onTimeStringChange}
-          {...timeInputFocusProps}
-          error={!parseSuccess && !isTimeFocused && `Invalid time, please enter a value like "11:00 AM"`}
-        />
+      {(props.includeTime ?? (config as any).includeTime) && (
+        <Popover
+          active={popoverActive}
+          onClose={() => setPopoverActive(false)}
+          activator={
+            <TextField
```

Comment:
Right now, a user has no way of passing in their own custom props to the Date or time inputs. Perhaps some props called `datePickerProps?: Partial<DatePickerProps> ` and `timePickerProps: Partial<TextFieldProps>` could be added so that this would be possible. 


packages/react/src/auto/polaris/submit/PolarisSubmitResultBanner.tsx
^ An example of something similar